vector/src/main/java/im/vector/riotredesign/features/home/room/detail/timeline/action/MessageActionsBottomSheet.kt

```diff
@@ -117,36 +117,36 @@ class MessageActionsBottomSheet : BaseMvRxBottomSheetDialog() {
     }
 
     override fun invalidate() = withState(viewModel) {
-        senderNameTextView.text = it.senderName
-        messageBodyTextView.text = it.messageBody
-        messageTimestampText.text = it.ts
-
-        GlideApp.with(this).clear(senderAvatarImageView)
-        if (it.senderAvatarPath != null) {
-            GlideApp.with(this)
-                    .load(it.senderAvatarPath)
-                    .circleCrop()
-                    .placeholder(AvatarRenderer.getPlaceholderDrawable(requireContext(), it.userId, it.senderName))
-                    .into(senderAvatarImageView)
+        if (it.showPreview) {
+            bottom_sheet_message_preview.isVisible = true
+            senderNameTextView.text = it.senderName
+            messageBodyTextView.text = it.messageBody
+            messageTimestampText.text = it.ts
+
+            GlideApp.with(this).clear(senderAvatarImageView)
```

Comment:
Why not using AvatarRenderer for the whole stuff?

Reply:
yes, thx